src/dashboard/src/media/js/ingest.js

```diff
@@ -295,25 +290,28 @@ $(function()
               });
             };
 
+          // Upload DIP to Archivists Toolkit
           // redict to object/resource mapping pages
-          if ('- Upload DIP to Archivists Toolkit' == $select.find('option:selected').text())
+          if ('f11409ad-cf3c-4e7f-b0d5-4be32d98229b' == $select.find('option:selected').val())
           {
             $('body').html('<h1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;' + gettext('Loading...') + '</h1>');
             window.location.href = '/ingest/' + this.model.sip.get('uuid') + '/upload/atk/';
           }
 
+          // Upload DIP to ArchivesSpace
           // redirect to object/resource mapping pages
-          if ('- Upload DIP to ArchivesSpace' == $select.find('option:selected').text())
+          if ('3572f844-5e69-4000-a24b-4e32d3487f82' == $select.find('option:selected').val())
           {
             $('body').html('<h1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;' + gettext('Loading...') + '</h1>');
             window.location.href = '/ingest/' + this.model.sip.get('uuid') + '/upload/as/';
           }
 
+          // Upload DIP to AtoM/Binder
           // If no identifier for the AtoM or Binder SWORD V1 deposit endpoint
           // provided at start of transfer, display a modal dialog to request
           // such here.
           if (
-            '- upload-qubit_v0.0' == $select.find('option:selected').text()
+            '0fe9842f-9519-4067-a691-8a363132ae24' == $select.find('option:selected').val()
```

Comment:
Related to the above. This UUID is a chain not a chain-link, so it might be useful to elaborate on that.